storage/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.urls import path
+
+from storage.views import MylistView, MylistMusicView
+
+urlpatterns =[
+    path('/mylist', MylistView.as_view()),
+    path('/mylist/<int:mylist_id>', MylistView.as_view()),
+    path('/mylistmusic/<int:mylist_id>', MylistMusicView.as_view()),
```

Comment:
엔드포인트의 URI가 혼란을 줄 수 있을 것 같습니다.
`/mylistmusic/1 `으로 해당 자원을 명확하게 알 수 있어야되는데 이렇게만 보면 `1`은 mylistmusic 데이터 중에 1번으로 보입니다.
그런데, 실제 로직은 mylist_id를 받고 있네요.

restful하게 수정해주세요~
```suggestion
    path('/mylistmusic/<int:mylist_id>', MylistMusicView.as_view()),
```

Reply:
반영했습니다.

/mylist/12/music

id가 12인 mylist 의 뮤직을 가져온다고 해석 가능한 restful 엔드포인트로 수정했습니다.